generators/client-2/templates/src/main/webapp/app/home/home.html

```diff
@@ -4,7 +4,7 @@
             <span class="hipster img-responsive img-rounded"></span>
         </div>
         <div class="col-md-8">
-            <h1 translate="home.title">Welcome, Java Hipster!</h1>
+            <h1 [jhi-translate]="'home.title' | translate">Welcome, Java Hipster!</h1>
```

Comment:
@sendilkumarn the solution for this is too simple and stupid, we just need to think from a stupid perspective 😉 
```js
import { Component } from '@angular/core';

@Component({
  selector: '[jhi-translate],[jhiTranslate]',
  template: '{{ key | translate }}',
  inputs: ['key:jhi-translate']
})
export class JhiTranslate {

  private key: string;
  //TODO add support to pass translate-values and for translate-compile

}
```